app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,39 @@
+module SessionsHelper
+	def log_in(user)
+		session[:user_id] = user.id
+	end
+	
+	def logged_in?
+		!current_user.nil?
+	end
+
+  def log_off
+    forget(current_user)
+		session.delete(:user_id)
+		@current_user = nil
+	end
+
+	def current_user
+		if (user_id = session[:user_id])
+		  @current_user ||= User.find_by(id: user_id)
+		elsif (user_id = cookies.signed[:user_id])
+      user = User.find_by(id: user_id)
+		if user && user.authenticated?(cookies[:remember_token])
+		  log_in user
+			@current_user = user
```

Comment:
- [ ] You should use the #current_user=(user) method to set user ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/current_user_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))